services/search/utils.py

```diff
@@ -1,7 +1,38 @@
-import requests  # pylint: disable=import-error
+import logging
+from requests_futures.sessions import FuturesSession  # pylint: disable=import-error
 import constants
 
 
+class SynonymExtractor:
+    """
+    Wrapper class for requests_futures methods
+    """
+    def __init__(self, max_synonyms):
+        self.max_synonyms = max_synonyms
+        self.session = FuturesSession()
+
+    def create_conc_reqs(self, word_arr):
+        """
+        create an asynchronous request list using requests-futures
+        Attributes:
+            word_arr: A list of words which each is a different request
+            session: A requests-futures session
+        Return:
+            A list of requests for every word
+        """
+        reqs = []
+        for word in word_arr:
+            reqs.append(self.session.get(constants.SPANISH_API_URL+word))  # pylint: disable=no-member
+
+        return reqs
+
+    def req_to_json(self, json):
+        """
+        Returns json from request
+        """
+        return json.result().json()
```

Comment:
nit: The naming here is a little confusing. This method is actually getting the response and converting it to JSON, so maybe a better name would be something like `get_response_json`. 

Also, the first arg is called `json` but it's actually a `request`. Consider giving it a less confusing name.